src/Cake.Core/IO/IDirectory.cs

```diff
@@ -32,7 +32,8 @@ public interface IDirectory : IFileSystemInfo
         /// Deletes the directory.
         /// </summary>
         /// <param name="recursive">Will perform a recursive delete if set to <c>true</c>.</param>
-        void Delete(bool recursive);
+        /// <param name="force">Will delete read-only files if set to <c>true</c></param>
+        void Delete(bool recursive, bool force);
```

Comment:
If doing so that would involve [IFile::Delete()](https://github.com/DixonDs/cake/blob/e515d2408311e5b8b5d0736e33f566f9e7212f11/src/Cake.Core/IO/IFile.cs#L42) too.